code/game/machinery/engi_points_vendor

```diff
@@ -0,0 +1,35 @@
+/*	
+	var/static/list/prize_list = list(
+		new /datum/GBP_equipment("Tendie",				/obj/item/weapon/reagent_containers/food/snacks/nugget,								50,		1),
+		new /datum/GBP_equipment("Cigar",				/obj/item/clothing/mask/cigarette/cigar/havana,													50,		1),
+		new /datum/GBP_equipment("Fulton Beacon",		/obj/item/fulton_core,																							50,		1),
+		new /datum/GBP_equipment("Soap",				/obj/item/weapon/soap/nanotrasen,																				250,	1),
+		new /datum/GBP_equipment("Advanced Indoor Fulton Pack",			/obj/item/weapon/extraction_pack/advanced,					300,	1),
```

Comment:
They have lots of heavy machinery that would make this clutch in something like blob. For naughty antags I'm holding out for the day someone repeats what I did in the video and sends a timed canister full of burning plasma into a secure location. 

Also it could work quite well aboard the new ship. I agree they're kinda silly, I'll wait and see what others say and if they get used during testing.